api/envoy/extensions/filters/http/error_response/v3/error_response.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.error_response.v3;
+
+import "envoy/config/accesslog/v3/accesslog.proto";
+import "envoy/config/core/v3/base.proto";
+import "envoy/config/core/v3/substitution_format_string.proto";
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.error_response.v3";
+option java_outer_classname = "ErrorResponseProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: ErrorResponse]
+// Error Response Filter :ref:`configuration overview <config_http_filters_error_response>`.
+// [#extension: envoy.filters.http.error_Response]
+
+// The configuration to filter and change local response.
+message ResponseMapper {
+  // Filter to determine if this mapper should apply.
+  config.accesslog.v3.AccessLogFilter filter = 1 [(validate.rules).message = {required: true}];
+
+  // The new response status code if specified.
+  google.protobuf.UInt32Value status_code = 2 [(validate.rules).uint32 = {lt: 600 gte: 200}];
+
+  // The new error body text if specified. It will be used in the `%LOCAL_REPLY_BODY%`
+  // command operator in the `body_foramt`.
```

Comment:
> foramt

chcek yuor spleling

Reply:
looks like there's a typo in api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto's version of this comment, i'll fix it in our copy